pkgs/tools/security/hashcash/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchurl, openssl }:
+
+stdenv.mkDerivation rec {
+  name = "hashcash-${version}";
+  version = "1.22";
+
+  buildInputs = [ openssl ];
+
+  src = fetchurl {
+      url = "http://www.hashcash.org/source/hashcash-1.22.tgz";
```

Comment:
Please use two spaces for indentation. Also use `${version}` in the URL, so it can be updated more easily.

Reply:
Done in 6b8c09bffca82c1373c38586cac2032088f0e5bf